docs/csharp/write-safe-efficient-code.md

```diff
@@ -74,35 +74,35 @@ public struct Point3D
 {
     public Point3D(double x, double y, double z)
     {
-        this.X = x;
-        this.Y = y;
-        this.Z = z;
+        _x = x;
```

Comment:
This is a stylistic decision (both versions work). We don't make a recommendation in our current style guide. I'll merge this with this change.

Reply:
leaving it in the original state would cause CS0188 due to assigning via property instead to field directly.

Reply:
Ahh. Yes. When I looked at the code I missed that it was a `struct`. Thanks for the correction.